FrontEnd/src/components/Header/Navbar/Navbar.module.css

```diff
@@ -13,11 +21,48 @@
 
 .navbar-main-logo {
   display: flex;
-  width: 199px;
-  height: 24px;
+  align-items: center;
+  justify-content: space-between;
+  gap: 10px;
+  color: #000000;
+  width: 208px;
+  height: 30px;
+  font-size: 24px;
+  font-weight: 700;
 }
 
 .navbar-utility-bar {
   display: flex;
   gap: 24px;
+  height: 40px;
+}
+
+.burger-menu-wrapper {
+  display: none;
+}
+
+.menu-search-wrapper{
+  display: flex;
+  gap: 24px;
+}
+
+@media (max-width: 1200px) {
```

Comment:
I would recommend to create predefined mixings like

```
@mixin xl {
  @media (max-width: 1200px) {
    @content;
  }
}
```

And than use these pre-defined mixins
```
@include xl {
    width: 100%;
  }
```